cypress/e2e/assets_spec/asset_homepage.cy.ts

```diff
@@ -69,11 +69,11 @@ describe("Asset Tab", () => {
     );
     assetFilters.clickadvancefilter();
     assetFilters.clickslideoverbackbutton(); // to verify the back button doesn't clear applied filters
-    assetFilters.assertFacilityText("Dummy Facility 1");
-    assetFilters.assertAssetTypeText("INTERNAL");
-    assetFilters.assertAssetClassText("ONVIF");
-    assetFilters.assertStatusText("ACTIVE");
-    assetFilters.assertLocationText("Camera Locations");
+    assetFilters.assertFacilityText("Facility: Dummy Facility 1");
+    assetFilters.assertAssetTypeText("Asset Type: INTERNAL");
+    assetFilters.assertAssetClassText("Asset Class: ONVIF");
+    assetFilters.assertStatusText("Status: ACTIVE");
+    assetFilters.assertLocationText("Location: Camera Loc");
```

Comment:
@kshitijv256 the code is checking for whether the data is contained in the badge, don't keep an exact match of the data to verify it

Reply:
It is not working if not providing exact values here